presto-native-execution/presto_cpp/main/connectors/CMakeLists.txt

```diff
@@ -18,4 +18,4 @@ if(PRESTO_ENABLE_ARROW_FLIGHT_CONNECTOR)
 endif()
 
 target_link_libraries(presto_connectors presto_velox_expr_conversion
-                      velox_type_fbhive)
+                      velox_type_fbhive velox_clp_connector)
```

Comment:
Do we really need this? If so, let's alphabetize the list of libraries to link against.